sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,3 @@
+select *
+  from "customers"
+ where "isActive" = 'false';
```

Comment:
Don't quote booleans. This works only because Postgres aggressively converts data types.